tasks/main.yml

```diff
@@ -34,3 +34,32 @@
     enabled: true
   tags:
     - prometheus_run
+  when:
+    - not prometheus_docker_install
+
+- name: ensure container is up
+  docker_container:
+    name: prometheus
+    image: "{{ prometheus_docker_image }}:v{{ prometheus_version }}"
+    volumes:
+      - "prometheus_db:{{ __prometheus_db_container_dir }}:rw"
+      - "{{ prometheus_config_dir }}/prometheus.yml:/etc/prometheus/prometheus.yml:ro"
+      - "{{ prometheus_config_dir }}/conf.d:{{ prometheus_config_dir }}/conf.d:ro"
+      - "{{ prometheus_config_dir }}/rules:{{ prometheus_config_dir }}/rules:ro"
+      - "{{ prometheus_config_dir }}/file_sd:{{ prometheus_config_dir }}/file_sd:ro"
+    networks:
+      - name: prometheus
+    published_ports:
+      - "{{ prometheus_web_listen_port }}:{{ prometheus_web_listen_port }}"
```

Comment:
What if I set following:
```
prometheus_web_listen_port: 10000
prometheus_web_listen_address: 0.0.0.0:5000
```
?

Reply:
prometheus_web_listen_address is missleading I think but it that case it would be 10000. the variable `prometheus_web_listen_port` makes things more obvious about its usage where prometheus_web_listen_address does not only provide an address but a whole netsocket